src/api-schema/faction/basic.ts

```diff
@@ -47,6 +47,13 @@ const membersStructure: Structure = {
         "<user id>": fromStructure(memberStructure),
     },
 };
+const peaceStructure: Structure = {
+    id: "peace",
+    name: "Peace Treaties",
+    schema: {
+        "<faction id>": { type: EpochSeconds },
```

Comment:
We should probably get a description of what this field represents as well. It's clear for me, but it might not be clear for everyone instantly.